exercises/practice/dnd-character/.docs/instructions.md

```diff
@@ -0,0 +1,31 @@
+# Instructions
+
+For a game of [Dungeons & Dragons][dnd], each player starts by generating a character they can play with.
+This character has, among other things, six abilities; strength, dexterity, constitution, intelligence, wisdom and charisma.
+These six abilities have scores that are determined randomly.
```

Comment:
```suggestion
These six attributes have scores that are determined randomly.
```

Reply:
Yeah, it is all over the later rules now, so that dog has left the pound, so to speak.  I remember them as attributes and the adjustments made were to the abilities that one would perform, but it has been 40 years maybe...

Reply:
I understood the six attributes were adjusted by ability scores, though.  The attributes being those six core things, the ability scores being what is applied.  I am pretty sure it has been fine for the last 3 years though.